dataland-frontend/src/utils/Constants.ts

```diff
@@ -25,8 +25,7 @@ export const ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE = Object.values(DataTypeEnum).fi
 ) as Array<DataTypeEnum>;
 
 export const ARRAY_OF_FRAMEWORKS_WITH_UPLOAD_FORM = Object.values(DataTypeEnum).filter(
-  (frameworkName) =>
-    [DataTypeEnum.Sfdr as string, DataTypeEnum.Sme as string, DataTypeEnum.P2p as string].indexOf(frameworkName) === -1,
+  (frameworkName) => [DataTypeEnum.Sme as string, DataTypeEnum.P2p as string].indexOf(frameworkName) === -1,
```

Comment:
I have written this a long time ago, and now that I see it again, I think it could be just:

`export const ARRAY_OF_FRAMEWORKS_WITH_UPLOAD_FORM = [DataTypeEnum.Sme, DataTypeEnum.P2p .... etc.]`

Same for `ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE`